src/store/fileUploadStore.js

```diff
@@ -110,6 +119,16 @@ const mutations = {
 	setAttachmentFolder(state, attachmentFolder) {
 		state.attachmentFolder = attachmentFolder
 	},
+
+	// Sets the total size of the file in bytes
+	setProgressTotal(state, { uploadId, index, progressTotal }) {
+		Vue.set(state.uploads[uploadId].files[index], 'progressTotal', progressTotal)
```

Comment:
Rather than using `Vue.set` I would create and initialize the attribute when the object is created in `addFileToBeUploaded` (there the size can be got from `file.size`) so here it is just a matter of using a simple assignment.

Moreover that would also prevent getting a NaN when calling `uploadProgress` before the attributes are actually set.

And if I am not mistaken this does not store a progress, which would be something between 0 and 100, but the file size, so I would rename `progressTotal` to `totalSize`.